miasm2/jitter/jitcore_python.py

```diff
@@ -120,23 +121,25 @@ def myfunc(cpu):
                 # Manage resulting address
                 if isinstance(ad, m2_expr.ExprInt):
                     return ad.arg.arg
-                elif isinstance(ad, m2_expr.ExprId):
-                    cur_label = ad.name
+                elif isinstance(ad, m2_expr.ExprLoc):
+                    cur_loc_key = ad.loc_key
+                    if cur_loc_key not in known_loc_keys:
+                        return cur_loc_key
                 else:
                     raise NotImplementedError("Type not handled: %s" % ad)
 
-        # Associate myfunc with current label
-        self.lbl2jitbloc[label.offset] = myfunc
+        # Associate myfunc with current loc_key
+        self.lbl2jitbloc[self.ir_arch.symbol_pool.loc_key_to_offset(loc_key)] = myfunc
```

Comment:
An assert that `self.ir_arch.symbol_pool.loc_key_to_offset(loc_key)` is not None may be a good thing

Reply:
done